<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSetup</class>
 <widget class="QWidget" name="NetworkSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #17394A</string>
  </property>
  <widget class="QPushButton" name="networkoptionsnextButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="networkoptionsdescriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>configure how OSMC connects to the web</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="networkoptionsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>networking</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="radiobuttongroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(240, 240, 240);
border: 0px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="wirelessconnectionradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Souce Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>wireless connection</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wiredconnectionradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Souce Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>wired connection</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="advancednetworkingcheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>configure network manually</string>
   </property>
  </widget>
  <zorder>radiobuttongroupBox</zorder>
  <zorder>networkoptionsnextButton</zorder>
  <zorder>networkoptionsdescriptionLabel</zorder>
  <zorder>networkoptionsLabel</zorder>
  <zorder>advancednetworkingcheckBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
